ark/flowsom/preprocess.py

```diff
@@ -0,0 +1,96 @@
+import os
+
+import numpy as np
```

Comment:
Lets change the directory name to be phenotyping, rather than FlowSOM, since we won't actually be using the package directly